README.md

```diff
@@ -39,22 +38,24 @@ For science runs, you can also obtain a specific tag or branch _only_ (as oppose
 ```
 git clone -b v17.9.1 --single-branch git@github.com:GEOS-ESM/GEOSldas.git
 ```
-
+Helpful tip: You can speed up this step by applying the following, one-time `mepo` configuration change:
+```
+mepo config set clone.partial blobless
+```
+With this configuration change, the size of the initial clone will be reduced, and additional blobs are fetched later when needed.
 
 ### Step 3: Build the Model
 
 To build the model in a single step, do the following from a head node:
 ```
 cd ./GEOSldas
-parallel_build.csh
+./parallel_build.csh
 ```
 This checks out all the external repositories of the model (albeit only on the first run, [see subsection on mepo below](#mepo)!) and then builds and installs the model. 
 
-At **NCCS**, the default is to build GEOSldas on SLES12 (Skylake or Cascade Lake nodes); to build GEOSldas on SLES15 (Milan nodes), use `parallel_build.csh -mil`.
+The resulting model build is found in `build-Release/`, and the installation is found in `install-Release/`, with setup scripts like `ldas_setup` in `install-Release/bin/`.
```

Comment:
@weiyuan-jiang: I edited this paragraph and the next (https://github.com/GEOS-ESM/GEOSldas/pull/796/commits/1dba0d7e786250f5279f26c4b539b1cf2905fcbe) based on the build of the LDAS nightly tests.  Please verify if this matches what you get when you build the model.